pkg/latte/intent_login.go

```diff
@@ -148,7 +149,30 @@ func (i *IntentLogin) GetEffects(ctx context.Context, deps *workflow.Dependencie
 }
 
 func (i *IntentLogin) OutputData(ctx context.Context, deps *workflow.Dependencies, w *workflow.Workflow) (interface{}, error) {
-	return nil, nil
+	verifiedAuthns, err := i.getVerifiedAuthenticators(w)
+	if err != nil {
+		return nil, err
+	}
+	if len(verifiedAuthns) == 0 {
+		return nil, nil
+	}
+	userID := verifiedAuthns[0].UserID
```

Comment:
Why do we just use `i.Identity.UserID`?

Reply:
I think they are the same, but we should check `len(verifiedAuthns)` to ensure the user passed at least one step before including the email in response.